src/interface/types.rs

```diff
@@ -30,7 +30,7 @@ pub struct FSData {
 //redefining the StatData struct in this file so that we maintain flow of
 // program derive eq attributes for testing whether the structs equal other
 // statdata structs from stat/fstat
-#[derive(Eq, PartialEq, Default)]
+#[derive(Eq, PartialEq, Default, Debug)]
```

Comment:
are we printing statData anywhere?

Reply:
i found it handy to be able to print the statdata while writing this up, thought it would be useful to have as an ability. Though, I understand since it isnt being actively printed anywhere, this isnt required. Shall I remove this?

Reply:
yaa lets remove that